ivy/functional/frontends/jax/numpy/statistical.py

```diff
@@ -145,3 +146,82 @@ def max(a, axis=None, out=None, keepdims=False, where=None):
 
 
 amax = max
+
+
+@to_ivy_arrays_and_back
+def average(a, axis=None, weights=None, returned=False, keepdims=False):
+
+    # canonicalize_axis to ensure axis or the values in axis > 0
+    if axis is None:
+        pass
+    elif isinstance(axis, int):
+        pass
```

Comment:
Perhaps we don't need these two conditional branches? Could we remove these

Reply:
I have remove the conditional branches and tested it based on the currently written test